CHANGELOG.md

```diff
@@ -0,0 +1,11 @@
+# Change Log
+
+All notable changes to this project will be documented in this file.
+See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
+
+## [2.0.3](https://github.com/openwallet-foundation-labs/sd-jwt-js/compare/v2.0.2...v2.0.3) (2024-02-20)
```

Comment:
I assume that this changelog generated automatically, is it right?
It might be needed to fix manually as we discuss in #44.  To 0.x

Reply:
I think we should remove the generated one from this PR. As mentioned above, I don't know what's the best way to generate the changelog that should be part of the github repo. It should be relatively easy to use the release mechanism of github

Reply:
@cre8 I think we might need to reset all the tags in this repo when we deploy to @sd-jwt/*

Reply:
I guess release mechanism of github is easiest way :D.